video2dataset/subsamplers/optical_flow_subsampler.py

```diff
@@ -120,15 +217,20 @@ class OpticalFlowSubsampler:
         fps (int): The target frames per second. Defaults to -1 (original FPS).
     """
 
-    def __init__(self, detector="cv2", fps=-1, params=None, downsample_size=None, dtype=np.float16):
+    def __init__(self, detector="cv2", fps=-1, args=None, downsample_size=None, dtype=np.float16):
         if detector == "cv2":
-            if params:
-                pyr_scale, levels, winsize, iterations, poly_n, poly_sigma, flags = params
+            if args:
+                pyr_scale, levels, winsize, iterations, poly_n, poly_sigma, flags = args
                 self.detector = Cv2Detector(
                     pyr_scale, levels, winsize, iterations, poly_n, poly_sigma, flags, downsample_size=downsample_size
                 )
             else:
                 self.detector = Cv2Detector(downsample_size=downsample_size)
+        elif detector == "raft":
+            assert args is not None
+            if not isinstance(args, AttrDict):
+                args = AttrDict(args)
```

Comment:
this is required to make RAFT work (it expects command line args...) i can either refactor RAFT on my fork or include this quick fix (i prefer the latter)

Reply:
nope this is good